### DISCUSSION TITLE
Is this working anymore?

### DISCUSSION THREAD

**Discussion by Redeorhcs** [rodit/SnapMod#295] [Q&A/Support]
I have a hard time getting any answers from anybody on the exposure pository, telegram check, but looking through the discussions here on GitHub, it looks like this mod no longer works. Someone reported that snapchats server side won't let you run anything under version 12 anymore? Is this true? 11.96 isn't that outdated! There is also a lot of discussion about roddit returning to the project. Did he mention stepping away from it?

**Comment by GoatBan** [comment]
Everytging wor fine @RevealedSoulEven got a forked version for 11.92 with some broken features,, olders version are also working. 
From what I understand Rodit still a student so he prolly not have a lot of time

**Comment by RevealedSoulEven** [comment]
Actually in 11.92 everything is working fine and that's same for the latest
12.(something) too. Just splitting snap isn't working in 12.x but it's good

**Comment by Redeorhcs** [comment]
So it's worth a try to just download and use this module on version
12.13.0.33?

**Comment by RevealedSoulEven** [comment]
See my github page, you'll find the version along with the apk
